### ISSUE TITLE
sidekiq-cloudwatchmetrics incompatible with sidekiq 6.5

### ISSUE THREAD

**Issue by nealblomfield** [sj26/sidekiq-cloudwatchmetrics#25]
Upgrading to sidekiq 6.5.0 / sidekiq-pro 5.5.0 / sidekiq-ent 2.5.0 breaks sidekiq-cloudwatchmetric with the following error.

<img width="652" alt="Screen Shot 2022-06-15 at 10 33 32 AM" src="https://user-images.githubusercontent.com/104747474/173700365-8f27f43f-08b3-4b62-9c92-16ef500487ff.png">

See https://github.com/mperham/sidekiq/issues/5372 for similar issues raised by a number of other gems that depend on sidekiq and `require sidekiq/util`.

**Comment by johan-smits** [comment]
See a fix for this: https://github.com/LeftClickBV/sidekiq-cloudwatchmetrics/commit/42f8fa5f14141e2fd4a0a22f32eb6cb4296a6947

**Comment by worldoffarming** [comment]
I am using sidekiq 6.5.3 on production and was trying to use coudwatchmetrics
I am facing the same issue after configuring it and running rails server

```
$ bundle exec rails s
/Users/user/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require': cannot load such file -- sidekiq/util (LoadError)
	from /Users/user/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:17:in `require'
	from /Users/user/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/sidekiq-cloudwatchmetrics-2.2.0/lib/sidekiq/cloudwatchmetrics.rb:5:in `<main>'
	from /Users/user/.rbenv/versions/3.0.4/lib/ruby/gems/3.0.0/gems/bootsnap-1.13.0/lib/bootsnap/load_path_cache/core_ext/kernel_require.rb:32:in `require .... 

```
can we merge @johan-smits 's fix ?

**Comment by sj26** [comment]
Thanks for the report, and sorry for taking so long to get to this. I had notifications switched off (which has also been fixed). Compatibility has been fixed in 47091a9. I'll be doing some cleanup, then releasing a new version of the gem.]

**Comment by ngouy** [comment]
THanks a log @sj26 🙏
